examples/svdiff/README.md

```diff
@@ -0,0 +1,298 @@
+# SVDiff
```

Comment:
@mkshing I have performed several edits to the README to help make it more readable for the users. Could you have a look?

Reply:
@sayakpaul LGTM! Thank you!